src/Persistence/Sql.php

```diff
@@ -620,6 +620,10 @@ public function action(Model $model, $type, $args = [])
                 $this->initQueryConditions($model, $query);
                 $this->setLimitOrder($model, $query);
 
+                if ($model->loaded()) {
+                    $query->where($model->id_field, $model->id);
```

Comment:
action materializes/render, so this makes sense (like `action('update')` uses the current ID)

before merge, confirm, that all uses of `action('field')` expect restricted field - does it make any sense to be used somewhere without the restriction to id?

Reply:
Not 100% sure about this. Maybe there is some use-case where this is supposed to generate
`SELECT field FROM table` without ID condition even if model is already loaded.

Reply:
That is why I messages you about these changes - I think this is ok, as user can easily unload (or even clone + unload) and then call this method...

Reply:
No issues have been observed so far in testing, we will see in realtime use but this is true - model unload should be the solution if such usage exists